Makefile.inc

```diff
@@ -1,5 +1,4 @@
 , := ,
-_using = $(subst $(,), ,$(using))
```

Comment:
This still can;t be removed as it;s being used by consuming projects.

They will have to be updated to use `USING` and only then can this be removed

Reply:
Or we need to step way back re-evaluate again how we populate `SETTINGS` i.e Re-verting / re-working 
https://github.com/submariner-io/shipyard/commit/a1ab011f2271478a694aeba864099f09f37ff93d

Please let me know what the correct way forward is here

Reply:
It makes sense to me to keep the var as a "shadow" (`_using = ${USING}`) until it can be removed.

Reply:
Also

> /opt/shipyard/scripts/deploy.sh --cable_driver libreswan --deploytool helm --settings /go/src/github.com/submariner-io/submariner/.shipyard.e2e.yml
2022-04-12T01:42:59.0911727Z Running with: globalnet='false', deploytool='helm', deploytool_broker_args='', deploytool_submariner_args='', settings='/go/src/github.com/submariner-io/submariner/.shipyard.e2e.yml', timeout=5m, image_tag=local, cable_driver=libreswan, service_discovery=false

https://github.com/submariner-io/shipyard/runs/5982624162?check_suite_focus=true

Reply:
Problem is all of these "using" cases in the consuming e2e are actually parsed on the Shipyard level, e.g. `using=lighthouse` in Submariner is not tested and that would break:
https://github.com/submariner-io/submariner-operator/blob/devel/Makefile#L61-L63

Reply:
Yeah that makes sense, the code path for `using` in the consuming tests isn't the same as if we were running in a consuming project directly.

Reply:
> It makes sense to me to keep the var as a "shadow" (_using = ${USING}) until it can be removed. 

Done and tested manually, all seems to work!